gulp.d/tasks/build.js

```diff
@@ -48,9 +49,9 @@ module.exports = (src, dest, preview) => () => {
         },
       },
     ]),
-    postcssNesting,
-    // NOTE because css vars are our theme switching mechanism we must preserve them with preserve: true
-    postcssVar({ preserve: true }),
+    postcssNested,
+    postcssAdvancedVars,
+    postcssVar({ preserve: preview }),
```

Comment:
after upgrading postcss-custom-variables and npm this works properly now when preserve is false for the production bundle